go/rtl/settings.go

```diff
@@ -37,3 +44,32 @@ func NewSettingsFromFile(file string, section *string) (ApiSettings, error) {
 	return s, err
 
 }
+
+func NewSettingsFromEnv() (ApiSettings, error) {
+	settings := defaultSettings
+
+	if v, present  := os.LookupEnv(baseUrlEnvKey); present {
+		settings.BaseUrl = v
+	}
+	if v, present  := os.LookupEnv(apiVersionEnvKey); present {
+		settings.ApiVersion = v
+	}
+	if v, present  := os.LookupEnv(verifySslEnvKey); present {
+		s := strings.ToLower(v)
+		settings.VerifySsl = s == "true" || s == "t" || s == "1" || s == "y" || s == "yes"
```

Comment:
fwiw other SDKs have either an `isBool()` or `defaultBool()` type of helper function. In this case, I think we want to default `settings.VerifySSL` to `true`

Reply:
Oh good catch! I will fix.

Reply:
I just reread the spec. My logic is correct. It already has setting default to true. Only if it is set does it change.
```
true, t, yes, y, or 1 (case insensitive) to enable SSL verification. Any other value is treated as false. Defaults to true if not set.
```